docs/os-ref.md

```diff
@@ -4,615 +4,552 @@ title: aragonOS reference documentation
+First of all, you can obtain the default ACL instance for a Kernel with:
 
 ```solidity
-acl = ACL(kernel.acl())
+ACL acl = ACL(kernel.acl());
 ```
 
 Then you can execute the following actions:
 
-#### Create Permission
+**Create Permission**
 
 ```solidity
-acl.createPermission(address entity, address app, bytes32 role, address manager)
+acl.createPermission(address entity, address app, bytes32 role, address manager);
 ```
 
-`createPermission()` will fail if that permission has pre-existing permission instances.
+- **entity**: entity to hold the permission.
+- **app**: app whose role will be allowed through the permission
+- **role**: role to allow
+- **manager**: manager of the role's permission instances
+
+> **Warning**
+>
+> `createPermission()` will fail if that role has pre-existing permission instances or a permission manager set.
 
-This action is identical to [`grantPermission()`](#grant-permission) except it allows the creation of a new permission if it doesn’t exist yet.
+Grants `role` in `app` for an `entity` and set `manager` as the manager of the role's permission instances.
 
-A role in the ACL protects access to `createPermission()` as this important function could be used in malicious ways. When the Kernel is initialized, it also creates the permission that grants the initializing address the ability to create new permissions.
+This action is identical to `grantPermission()` except it allows the creation of the first permission instance of a role.
 
-Note that creating permissions is made mandatory by the ACL: all actions requiring yet-to-be-created permissions are disallowed by default. Any permission checks on non-existent permissions are failed automatically.
+> **Note**
+>
+> Creating permissions is mandatory for apps to work. Any permission checks on non-existent permissions are failed automatically.
 
-#### Grant Permission
+**Grant Permission**
 
 ```solidity
-acl.grantPermission(address entity, address app, bytes32 role)
+acl.grantPermission(address entity, address app, bytes32 role);
 ```
 
-Grants `role` in `app` for an `entity`. Only callable by the `manager` of a certain permission. This `entity` would then be allowed to call all actions that their `role` can perform on that particular `app` until the permission manager revokes their role with [`revokePermission()`](#revoke-permission).
+> **Warning**
+>
+> Only callable by the `manager` of the role's permission instances.
+
+Grants `role` in `app` for an `entity`. This `entity` would then be allowed to call all actions that their `role` can perform on that particular `app` until the permission manager revokes their role with `revokePermission()`.
 
-The `grantPermission()` action doesn’t require protection with the ACL because an entity can only make changes to a permission if it is the permission's `manager`.
+This action is identical to `grantPermission()` except it can only be used by the permission manager of the role.
```

Comment:
I think this isn't necessary here

Reply:
😅 oops, forgot to change it to read `createPermission`. Fixed in 7b743b6